azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -62,13 +62,7 @@ jobs:
   - task: Gradle@2
     displayName: Run Unit tests
     inputs:
-      tasks: msal:testLocalDebugUnitTest -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}} --tests *com.microsoft.identity.client*
-      javaHomeSelection: $(BuildParameters.javaHomeSelection)
-      jdkVersion: 1.11
-  - task: Gradle@2
-    displayName: Run NativeAuth Unit tests
-    inputs:
-      tasks: msal:testLocalDebugUnitTest -PuseMockApiForNativeAuth=true -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}} --tests *com.microsoft.identity.nativeauth*
+      tasks: msal:testLocalDebugUnitTest -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}}
```

Comment:
Same here - our test suite heavily depends on the mock API. We can't disable it, because tests will fail. Furthermore I don't understand why this is necessary.

Reply:
I see you've replaced this with setting this in the test 👍 I'd like to see the results of the tests in the CI before I approve.